# @generated
load("@antlir//antlir/antlir2/package_managers/dnf/rules:rpm.bzl", "rpm")
load("@antlir//antlir/antlir2/package_managers/dnf/rules:xml.bzl", "xml")
rpm(name='0-3.3.35-2.el9.x86_64-f9fe5', rpm_name='setroubleshoot-server', epoch=0, version='3.3.35', release='2.el9', arch='x86_64', url='https://mirror.facebook.net/centos-composes/9/CentOS-Stream-9-20251004.0/compose/AppStream/x86_64/os/Packages/setroubleshoot-server-3.3.35-2.el9.x86_64.rpm', xml=':0-3.3.35-2.el9.x86_64-f9fe5--xml', sha1=None, sha256='f9fe5cffb30ab04e882aece4f907411d0e1727a5506e588cfc57089efc486a90', visibility=['//snapshot/rpm/centos9-compose/...'])
xml(name='0-3.3.35-2.el9.x86_64-f9fe5--xml', primary='<package type="rpm">\n  <name>setroubleshoot-server</name>\n  <arch>x86_64</arch>\n  <version epoch="0" ver="3.3.35" rel="2.el9"/>\n  <checksum type="sha256" pkgid="YES">f9fe5cffb30ab04e882aece4f907411d0e1727a5506e588cfc57089efc486a90</checksum>\n  <summary>SELinux troubleshoot server</summary>\n  <description>Provides tools to help diagnose SELinux problems. When AVC messages\nare generated an alert can be generated that will give information\nabout the problem and help track its resolution. Alerts can be configured\nto user preference. The same tools can be run on existing log files.</description>\n  <packager>builder@centos.org</packager>\n  <url>https://gitlab.com/setroubleshoot/setroubleshoot</url>\n  <time file="1752549883" build="1751991501"/>\n  <size package="365530" installed="1355282" archive="1382040"/>\n  <location href="Packages/f9fe5cffb30ab04e882aece4f907411d0e1727a5506e588cfc57089efc486a90/setroubleshoot-server-0:3.3.35-2.el9.x86_64.rpm"/>\n  <format>\n    <rpm:license>GPL-2.0-or-later</rpm:license>\n    <rpm:vendor>CentOS</rpm:vendor>\n    <rpm:group>Unspecified</rpm:group>\n    <rpm:buildhost>x86-03.stream.rdu2.redhat.com</rpm:buildhost>\n    <rpm:sourcerpm>setroubleshoot-3.3.35-2.el9.src.rpm</rpm:sourcerpm>\n    <rpm:header-range start="36360" end="66385"/>\n    <rpm:provides>\n      <rpm:entry name="config(setroubleshoot-server)" flags="EQ" epoch="0" ver="3.3.35" rel="2.el9"/>\n      <rpm:entry name="group(setroubleshoot)"/>\n      <rpm:entry name="python3.9dist(setroubleshoot)" flags="EQ" epoch="0" ver="3.3.33"/>\n      <rpm:entry name="python3dist(setroubleshoot)" flags="EQ" epoch="0" ver="3.3.33"/>\n      <rpm:entry name="setroubleshoot-server" flags="EQ" epoch="0" ver="3.3.35" rel="2.el9"/>\n      <rpm:entry name="setroubleshoot-server(x86-64)" flags="EQ" epoch="0" ver="3.3.35" rel="2.el9"/>\n      <rpm:entry name="user(setroubleshoot)"/>\n    </rpm:provides>\n    <rpm:requires>\n      <rpm:entry name="/bin/sh" pre="1"/>\n      <rpm:entry name="/bin/sh"/>\n      <rpm:entry name="/usr/bin/python3"/>\n      <rpm:entry name="audit" flags="GE" epoch="0" ver="3.0.1"/>\n      <rpm:entry name="audit-libs-python3"/>\n      <rpm:entry name="dbus"/>\n      <rpm:entry name="initscripts-service"/>\n      <rpm:entry name="libauparse.so.0()(64bit)"/>\n      <rpm:entry name="libcap-ng.so.0()(64bit)"/>\n      <rpm:entry name="libdbus-1.so.3()(64bit)"/>\n      <rpm:entry name="libdbus-1.so.3(LIBDBUS_1_3)(64bit)"/>\n      <rpm:entry name="libselinux-python3" flags="GE" epoch="0" ver="2.1.5" rel="1"/>\n      <rpm:entry name="libselinux.so.1()(64bit)"/>\n      <rpm:entry name="libselinux.so.1(LIBSELINUX_1.0)(64bit)"/>\n      <rpm:entry name="libxml2-python3"/>\n      <rpm:entry name="policycoreutils-python-utils"/>\n      <rpm:entry name="polkit"/>\n      <rpm:entry name="python(abi)" flags="EQ" epoch="0" ver="3.9"/>\n      <rpm:entry name="python3-dasbus"/>\n      <rpm:entry name="python3-dbus"/>\n      <rpm:entry name="python3-gobject-base" flags="GE" epoch="0" ver="3.11"/>\n      <rpm:entry name="python3-six"/>\n      <rpm:entry name="rpm-python3"/>\n      <rpm:entry name="rtld(GNU_HASH)"/>\n      <rpm:entry name="setroubleshoot-plugins" flags="GE" epoch="0" ver="3.3.10"/>\n      <rpm:entry name="systemd-python3" flags="GE" epoch="0" ver="206" rel="1"/>\n      <rpm:entry name="libc.so.6(GLIBC_2.34)(64bit)"/>\n    </rpm:requires>\n    <file>/etc/audit/plugins.d/sedispatch.conf</file>\n    <file>/etc/dbus-1/system.d/org.fedoraproject.SetroubleshootFixit.conf</file>\n    <file>/etc/dbus-1/system.d/org.fedoraproject.SetroubleshootPrivileged.conf</file>\n    <file>/etc/dbus-1/system.d/org.fedoraproject.Setroubleshootd.conf</file>\n    <file type="dir">/etc/setroubleshoot</file>\n    <file>/etc/setroubleshoot/setroubleshoot.conf</file>\n    <file>/usr/bin/sealert</file>\n    <file>/usr/sbin/sedispatch</file>\n    <file>/usr/sbin/setroubleshootd</file>\n  </format>\n</package>\n', filelists='<package pkgid="f9fe5cffb30ab04e882aece4f907411d0e1727a5506e588cfc57089efc486a90" name="setroubleshoot-server" arch="x86_64">\n  <version epoch="0" ver="3.3.35" rel="2.el9"/>\n  <file>/etc/audit/plugins.d/sedispatch.conf</file>\n  <file>/etc/dbus-1/system.d/org.fedoraproject.SetroubleshootFixit.conf</file>\n  <file>/etc/dbus-1/system.d/org.fedoraproject.SetroubleshootPrivileged.conf</file>\n  <file>/etc/dbus-1/system.d/org.fedoraproject.Setroubleshootd.conf</file>\n  <file type="dir">/etc/setroubleshoot</file>\n  <file>/etc/setroubleshoot/setroubleshoot.conf</file>\n  <file type="dir">/run/setroubleshoot</file>\n  <file>/usr/bin/sealert</file>\n  <file type="dir">/usr/lib/.build-id</file>\n  <file type="dir">/usr/lib/.build-id/67</file>\n  <file>/usr/lib/.build-id/67/739948222ce90f009628f6576ad87ac22900c8</file>\n  <file type="dir">/usr/lib/python3.9/site-packages/setroubleshoot</file>\n  <file type="dir">/usr/lib/python3.9/site-packages/setroubleshoot-3.3.33.dist-info</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot-3.3.33.dist-info/INSTALLER</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot-3.3.33.dist-info/METADATA</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot-3.3.33.dist-info/RECORD</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot-3.3.33.dist-info/REQUESTED</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot-3.3.33.dist-info/WHEEL</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot-3.3.33.dist-info/direct_url.json</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot-3.3.33.dist-info/top_level.txt</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/Plugin.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__init__.py</file>\n  <file type="dir">/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/Plugin.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/Plugin.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/__init__.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/__init__.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/access_control.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/access_control.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/analyze.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/analyze.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/audit_data.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/audit_data.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/avc_audit.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/avc_audit.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/config.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/config.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/email_alert.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/email_alert.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/errcode.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/errcode.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/html_util.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/html_util.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/rpc.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/rpc.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/rpc_interfaces.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/rpc_interfaces.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/server.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/server.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/serverconnection.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/serverconnection.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/signature.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/signature.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/util.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/util.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/uuid.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/uuid.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/xml_serialize.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/__pycache__/xml_serialize.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/access_control.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/analyze.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/audit_data.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/avc_audit.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/config.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/email_alert.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/errcode.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/html_util.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/rpc.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/rpc_interfaces.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/server.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/serverconnection.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/signature.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/util.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/uuid.py</file>\n  <file>/usr/lib/python3.9/site-packages/setroubleshoot/xml_serialize.py</file>\n  <file>/usr/lib/systemd/system/setroubleshootd.service</file>\n  <file>/usr/lib/sysusers.d/setroubleshoot.conf</file>\n  <file>/usr/lib/tmpfiles.d/setroubleshoot.conf</file>\n  <file>/usr/sbin/sedispatch</file>\n  <file>/usr/sbin/setroubleshootd</file>\n  <file>/usr/share/dbus-1/system-services/org.fedoraproject.SetroubleshootFixit.service</file>\n  <file>/usr/share/dbus-1/system-services/org.fedoraproject.SetroubleshootPrivileged.service</file>\n  <file>/usr/share/dbus-1/system-services/org.fedoraproject.Setroubleshootd.service</file>\n  <file type="dir">/usr/share/doc/setroubleshoot-server</file>\n  <file>/usr/share/doc/setroubleshoot-server/AUTHORS</file>\n  <file>/usr/share/doc/setroubleshoot-server/COPYING</file>\n  <file>/usr/share/doc/setroubleshoot-server/ChangeLog</file>\n  <file>/usr/share/doc/setroubleshoot-server/DBUS.md</file>\n  <file>/usr/share/doc/setroubleshoot-server/NEWS</file>\n  <file>/usr/share/doc/setroubleshoot-server/README</file>\n  <file>/usr/share/doc/setroubleshoot-server/TODO</file>\n  <file>/usr/share/locale/ar/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/as/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/bg/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/bn/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/bn_IN/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/ca/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/cs/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/da/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/de/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/el/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/en_GB/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/es/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/eu/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/fi/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/fr/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/fur/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/gl/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/gu/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/he/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/hi/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/hr/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/hu/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/it/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/ja/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/ka/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/kn/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/ko/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/mai/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/ml/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/mr/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/ms/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/nb/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/nds/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/nl/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/nn/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/or/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/pa/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/pl/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/pt/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/pt_BR/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/ru/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/si/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/sk/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/sr/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/sr@latin/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/sv/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/ta/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/te/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/th/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/tr/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/uk/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/zh_CN/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/locale/zh_TW/LC_MESSAGES/setroubleshoot.mo</file>\n  <file>/usr/share/man/man1/seapplet.1.gz</file>\n  <file>/usr/share/man/man8/sealert.8.gz</file>\n  <file>/usr/share/man/man8/sedispatch.8.gz</file>\n  <file>/usr/share/man/man8/setroubleshootd.8.gz</file>\n  <file>/usr/share/polkit-1/actions/org.fedoraproject.setroubleshootfixit.policy</file>\n  <file type="dir">/usr/share/setroubleshoot</file>\n  <file>/usr/share/setroubleshoot/SetroubleshootFixit.py</file>\n  <file>/usr/share/setroubleshoot/SetroubleshootPrivileged.py</file>\n  <file type="dir">/var/lib/setroubleshoot</file>\n  <file type="ghost">/var/lib/setroubleshoot/email_alert_recipients</file>\n  <file type="ghost">/var/lib/setroubleshoot/setroubleshoot_database.xml</file>\n</package>\n', other='<package pkgid="f9fe5cffb30ab04e882aece4f907411d0e1727a5506e588cfc57089efc486a90" name="setroubleshoot-server" arch="x86_64">\n  <version epoch="0" ver="3.3.35" rel="2.el9"/>\n  <changelog author="Petr Lautrbach &lt;lautrbach@redhat.com&gt; - 3.3.32-1" date="1690459200">- Always reset pending alarms when alarm(0) (rhbz#2112573)\n- \'imp\' module is deprecated in favor of \'importlib\' (rhbz#2224393)\n- Fix build with pip 23.1.2+\n- Remove dbus-glib-devel BR as it\'s only needed when compiled with seappletlegacy\n- Rename session bus name to org.fedoraproject.sealert\n- seapplet: wrap SEApplet() to try except\n- util.py: Add doctext test for build_module_type_cache()\n- Update translations</changelog>\n  <changelog author="Petr Lautrbach &lt;lautrbach@redhat.com&gt; - 3.3.35-1" date="1741867200">- Update tmpfiles.d config (bz#2346971)\n- Do not hardcode /var/lib/selinux as store_root\n- data: update app icon\n- Enable Georgian and Arabic (ar) languages in configure.ac\n- Update translations\n- Ship with contemporary app icon\n- Check that SELinux is enabled before running (rhbz#2178950)</changelog>\n  <changelog author="Vit Mojzis &lt;vmojzis@redhat.com&gt; - 3.3.35-2" date="1751371200">- audit_data: include syslog when needed (RHEL-39978)</changelog>\n</package>\n')
